tests/transactional/enable_selinux.pm

```diff
@@ -26,13 +27,25 @@ sub run {
 
     # install and enable SELinux if not done by default
     if (script_run('test -d /sys/fs/selinux && test -e /etc/selinux/config') != 0) {
-        assert_script_run('transactional-update setup-selinux');
+        trup_call('setup-selinux');
         upload_logs('/var/log/transactional-update.log');
         save_and_upload_log('rpm -qa', 'installed_pkgs.txt');
-        process_reboot(trigger => 1);
+        check_reboot_changes;
+        if (is_public_cloud) {
+            # Additional packages required for semanage
+            trup_call('pkg install policycoreutils-python-utils');
+            check_reboot_changes;
+            # allow ssh tunnel port (to openQA)
+            my $upload_port = get_required_var('QEMUPORT') + 1;
+            assert_script_run("semanage port -a -t ssh_port_t -p tcp $upload_port");
+            process_reboot(trigger => 1);
+        }
     }
 
     assert_script_run('selinuxenabled');
+    validate_script_output("getenforce", sub { m/Enforcing/ });
```

Comment:
Please test this change also in SLE Micro 5.1 and 5.2, where the default was not enforcing.